config/libc/bionic.in

```diff
@@ -5,6 +5,7 @@
 ## depends on ! WINDOWS && ! BARE_METAL
 ## depends on ARCH_ARM || ARCH_MIPS || ARCH_X86
 ## depends on EXPERIMENTAL
+## depends on OBSOLETE
```

Comment:
Kind of redundant (all versions of it are obsolete anyway) but perhaps a bit more visible to the user.

Also, you need to enable CT_OBSOLETE in the sample's configuration, or it will fail to build.

Reply:
> Kind of redundant (all versions of it are obsolete anyway) but perhaps a bit more visible to the user.

I think it makes it more obvious. Also prevents selecting bionic only to not be able to select any versions. 

> Also, you need to enable CT_OBSOLETE in the sample's configuration, or it will fail to build.

Yep done.